golang/README.md

```diff
@@ -0,0 +1,21 @@
+## Running the Example
+
+- [Go version 0.17.\*](https://go.dev/dl/) is required
```

Comment:
I think you mean version `1.17.*`? My local go version all start with 1

Reply:
Ah sorry it's typo. FIx that.